controllers/drcluster_controller.go

```diff
@@ -370,6 +376,24 @@ func (u *drclusterInstance) initializeStatus() {
 	}
 }
 
+func (u *drclusterInstance) getDRClusterDeployedStatus(drcluster *ramen.DRCluster) error {
+	mw, err := u.mwUtil.GetDrClusterManifestWork(drcluster.Name)
```

Comment:
The errors here appear in the resource conditions status, we can possibly word these better, some suggestions:

Line 381: "error fetching DRCluster ManifestWork"
Line 386: "missing DRCluster ManifestWork resource"
Line 391: "DRCluster ManifestWork is not in applied state"

Reply:
error messages fixed.